swap_meet/clothing.py

```diff
@@ -1,2 +1,24 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing (Item):
+    def __init__(self,condition = 0):
+        self.category = "Clothing"
+
+        super().__init__(self.category,condition)
```

Comment:
This works, but line 5 is duplicating work that is done by the Item constructor.  An alternative is to pass the string literal "Clothing" directly to the super constructor:
```suggestion
        super().__init__("Clothing",condition)
```